### DISCUSSION TITLE
Plz add enigma2 webif

### DISCUSSION THREAD

**Discussion by resToreHU** [iptv-org/community#17] [Ideas]
### Source

Http://webif

**Comment by HNJAMeindersma** [comment]
I don't think it would be possible to add this into the repo since Enigma2/OpenWebIf is usually a local hostname (inside a local area network). I'm also wondering if these receivers support exporting EPG data to a XMLTV format. Coincidentally I'll be experimenting with an OpenWebIf / OpenPLI / Enigma2 receiver in a couple of weeks.

**Comment by HNJAMeindersma** [comment]
I think you gonna need this: 
- https://github.com/Jalle19/openwebif-xmltv-converter
- https://pypi.org/project/OWi2Plex/

**Comment by resToreHU** [comment]
k
Thank you for addressing the issue!

I would like it to be included in the EPG grabber project.

Webgab++ has already solved it, but it would be nice if it appeared here as well.
Enigma2 plugin working openpli 8.3 and openatv 7.3 and 7.4.

Project name: epgexport

Thanks

**Comment by HNJAMeindersma** [comment]
> I would like it to be included in the EPG grabber project.

What would be your proposal regarding host specification for a possible Enigma2 XMLTV grabber that fits in with the current state and input arguments of the repo?

**Comment by freearhey** [comment]
1. Technically you can create a config that will work with OpenWebif. 
  I don't have access to enigma2 so I can't help with this, but according to this [thread](https://webgrabplus.com/comment/13104#comment-13104) parsing EPG from it is not very different from parsing a regular site.
  After that the finished config can be run directly using [epg-grabber](https://github.com/freearhey/epg-grabber).

2. As for adding such a config to [iptv-org/epg](https://github.com/iptv-org/epg) repository as it was correctly mentioned above it makes no sense as the list of available channels and IP address of TV box will be different for everyone.
